### DISCUSSION TITLE
Templates instead of Local

### DISCUSSION THREAD

**Discussion by RudraSen2** [LSPosed/MagiskOnWSALocal#408] [Q&A] [answered]
I have a question that instead of running this locally, like earlier, can this repository be provided as a template from settings? I'm not sure if templates also cause the same problem or not. Forks caused problems but I think that templates won't cause problems.

![image](https://user-images.githubusercontent.com/91307601/224535947-8fb7e605-1678-45de-b4fb-fc38abbe7763.png)

**Comment by Howard20181** [comment, marked as answer]
Setting the repository to template just encourages people not to fork, it doesn't stop forks.
And the previous repository being disabled wasn't caused by fork, it was caused by Actionsy being overused.
So marking the repository as a template now wouldn't be better.

**Comment by RudraSen2** [comment]
I know that actions were being overused by people who were forking the repository. So you are telling that Setting the template thing will also count the actions used by other people?

**Reply by Howard20181** [reply]
We now no longer offer an Actions-based build.
